Bottom: 22d5c5f48710b88acfe1daa41a318a85cb952581
Top:    1d2b3d29435a4e2483ed8bed898918b7b1364b9f
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-03-22 19:44:08 +0800

Refresh of vhost_net-prefetch-desc

---

diff --git a/drivers/vhost/vhost.c b/drivers/vhost/vhost.c
index 796de05..019ae79 100644
--- a/drivers/vhost/vhost.c
+++ b/drivers/vhost/vhost.c
@@ -2023,7 +2023,6 @@ int vhost_prefetch_desc_indices(struct vhost_virtqueue *vq,
 		++indices;
 		++last_avail_idx;
 	}
-	/* FIXME: update used ring here? together with batch dequing? */
 
 	/* Only get avail ring entries after they have been exposed by guest. */
 	smp_rmb();
